package.json

```
@@ -37,13 +37,13 @@
     "uuid": "3.x.x"
   },
   "devDependencies": {
-    "@pager/semantic-release-config": "1.x",
+    "@pager/semantic-release-config": "1.3.0",
```

```suggestion
    "@pager/semantic-release-config": "1.x.x",
```
Same for `mocha` and `semantic-release`.